### ISSUE TITLE
Request for API access: [Ammad]

### ISSUE THREAD

**Issue by anosidium** [sunnah-com/api#68]
* **Please tell us about yourself (include an email address):**
  

السلام عليكم ورحمة الله وبركاته

Ammad. I am an iOS Developer. I prefer not to write my e-mail address here to avoid unsolicited e-mails. I will be happy to disclose it by other means.

* **Your purpose in using this API:**
  

I would like to write an iOS app as seeing that the Quran.com has an app but the Sunnah.com doesn't. So it would be nice to have a companion app alongside with the Quran.com app.

* **API rate limits:**
  
   * **Maximum requests per second:**
   * **Maximum requests per day:**

Maximum requests per second seems logically if I cannot have offline data.

* **Is your use case better served by having an offline dump of hadith data or programmatic API access?**
  

I believe offline is better as the Quran.com works offline too. Maybe the app will do both, when there are new books, hadiths or corrections or updates then the app can fetch it and update the offline data.

* **What are the languages in which would you like hadith data?**
  

All languages that are available but I will prioritise English and Arabic first.

* **What programming language will your API client be in?**
  

I don't know yet but I will work it out once you give me permission. I intend to write the app natively in Swift. Not cross-platform language.

**Comment by ahadith** [comment]
Wa`alaykum ussalam warahmatullahi wabarakatuh

Please send us a contact message through the website with your name and email address.

We are happy to give you a key and you are welcome certainly to develop your own app, but I thought I would let you know that we are in the process of designing an official Sunnah.com app with all of our data and invite your help in doing so. We are still in the design phase.

**Comment by anosidium** [comment]
السلام عليكم ورحمة الله وبركاته

Done. Please let me know if you have received my message. I would love to get involved.

**Comment by aelbialy** [comment]
> Wa`alaykum ussalam warahmatullahi wabarakatuh
> 
> Please send us a contact message through the website with your name and email address.
> 
> We are happy to give you a key and you are welcome certainly to develop your own app, but I thought I would let you know that we are in the process of designing an official Sunnah.com app with all of our data and invite your help in doing so. We are still in the design phase.

Assalamu Alaikum,

I'm also an iOS developer and I was wondering if I can help with this initiative of creating an official Sunnah.com iOS app. I've sent a contact message through the website now.

**Comment by anosidium** [comment]
@ahadith السلام عليكم ورحمة الله وبركاته Have you received my e-mail? I've been waiting to hear from you or whoever. May I know more about the iOS app you are developing, please?

**Comment by aelbialy** [comment]
Yes same here. I haven't received anything or heard anything back from anyone. It would be nice to help contribute on this project
